src/PlusDataCollection/CMakeLists.txt

```diff
@@ -1645,6 +1646,107 @@ IF(PLUS_USE_INTERSONARRAYSDKCXX_VIDEO)
     )
 ENDIF()
 
+# --------------------------------------------------------------------------
+# Point Grey Spinnaker API support
+
+IF(PLUS_USE_SPINNAKER_VIDEO)
+  message("ABCD")
```

Comment:
I forget, is this where we typically put `find_package(Spinnaker...)`?

Reply:
We typically put the find_package() calls in the PlusBuild CMakeLists.txt